distributed/utils_test.py

```diff
@@ -631,6 +630,17 @@ def security():
     return tls_only_security()
 
 
+def _terminate_join(proc):
+    proc.terminate()
+    proc.join(timeout=30)
```

Comment:
How long does it typically take for a process to terminate? Is it worth splitting this up into "terminate all" then "join all" to speed things up?